### ISSUE TITLE
Terminal: focus border in HC theme could get some offset

### ISSUE THREAD

**Issue by bpasero** [Microsoft/vscode#19204]
The outline is very close to the actual content, which looks odd:

![image](https://cloud.githubusercontent.com/assets/900690/22250031/b76f747c-e245-11e6-8b6d-ad1cbf8afae7.png)

**Comment by Tyriar** [comment]
Switching to an absolute positioned `::before` over `outline` makes this nicer since `outline-offset` only accepts a single value for all edges:

![image](https://cloud.githubusercontent.com/assets/2193314/22387461/94aa5572-e490-11e6-92d1-e60692b7969f.png)

**Comment by bpasero** [comment]
It is better but still ugly because the outline is capped on the right:

![image](https://cloud.githubusercontent.com/assets/900690/23210759/553c4638-f8ff-11e6-96b6-aced13300563.png)

**Comment by Tyriar** [comment]
Should be fixed in https://github.com/Microsoft/vscode/issues/21333

**Comment by chrmarti** [comment]
- VSCode Version: Code - Insiders 1.10.0-insider (b2a55bd66eb60af11c60ddd61b7581538b216051, 2017-02-24T20:30:00.778Z)
- OS Version: Darwin x64 15.6.0

Not perfect, not sure if it can be improved easily @Tyriar:
![screen shot 2017-02-24 at 1 21 49 pm](https://cloud.githubusercontent.com/assets/9205389/23321432/60962e5e-fa94-11e6-8a86-37f787d756ab.png)

Note the 'd' going across the border on the right side.

**Comment by Tyriar** [comment]
Made it -5 instead of 5 as it was always meant to be. 

![image](https://cloud.githubusercontent.com/assets/2193314/23324977/763110e6-faa6-11e6-97c3-9fcb8ce22626.png)

**Comment by bpasero** [comment]
It is tricky:

![image](https://cloud.githubusercontent.com/assets/900690/23330046/192282e6-fb45-11e6-9712-a93ab018138d.png)

**Comment by Tyriar** [comment]
@bpasero that one should be fixed in the next build as the last fix https://github.com/Microsoft/vscode/commit/d41c48d46d1a1ce533d31f6c546d1d85bd2499a2 (11 hours ago) was shortly after the last build (14 hours ago).

**Comment by bpasero** [comment]
@Tyriar running "out of sources" I see the other issue again where the border is not all the way around:

![image](https://cloud.githubusercontent.com/assets/900690/23332952/f8b9f320-fb82-11e6-88c7-8d912333a04b.png)

**Comment by Tyriar** [comment]
@bpasero that's odd, are you sure you synced and rebuild? The screenshot in https://github.com/Microsoft/vscode/issues/19204#issuecomment-282434110 was out of sources after pushing and I just tried again on Linux and macOS with the same result after a `git clean -xfd`.
